specs/grafana_event_handling_in_monitoring_integration.adoc

```diff
@@ -0,0 +1,313 @@
+:imagesdir: ./images
```

Comment:
For this I guess I will have to create a new branch and submit a new PR. Should I do it?